### DISCUSSION TITLE
为什么我不能注册 Copilot？

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132870] [Copilot Conversations]
### Select Topic Area

Question

### Body

你好 Github 团队，
我在7月16日申请通过了学生包，7月20日发现权益下发通过，但是我在[注册 Copilot](/settings/copilot)时，提示了 `Your account is unable to sign up for Copilot. Please contact Support`

我去尝试了 Github 支持中的所有方法和网络上的方法，但还是不能用。  
![cannot-signup](https://github.com/user-attachments/assets/e0285e05-ce03-4528-96e7-9d0eabf7b779)
![approved](https://github.com/user-attachments/assets/93416a90-706c-4a92-9cb1-fd8391b231b5)

我去查看了计费账单，发现 Github Pro 可以使用，但是 Copilot 不行。

请问你们能让我注册吗？

**Comment by Akash1134** [comment]
Hi @ED-Builder , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)] very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
